org.openstreetmap.josm.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: org.openstreetmap.josm
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: josm
+rename-icon: josm
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+
+build-options:
+    env:
+        JAVA_HOME: /usr/lib/sdk/openjdk11/
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: josm
+    disabled: true
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/enable.sh
+      # - ant/bin/ant dist
+
+    sources:
+      - type: git
+        url:  git://github.com/openstreetmap/josm.git
+      - type: archive
+        url:  https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.2-bin.tar.xz
+        sha256: 361c8ad2ed8341416e323e7c28af10a8297170a80fdffba294a5c2031527bb6c
+        dest: ant
+
+  - name: josm-binary
+    buildsystem: simple
+    build-commands:
+        - install -D josm.sh /app/bin/josm
+        - install -D josm-snapshot-14382.jar /app/bin/josm.jar
+        - install -D org.openstreetmap.josm.desktop /app/share/applications/org.openstreetmap.josm.desktop
+        - install -D org.openstreetmap.josm.appdata.xml /app/share/metainfo/org.openstreetmap.josm.appdata.xml
+        - install -D josm.svg /app/share/icons/hicolor/scalable/apps/josm.svg
+        
+    sources:
+      - type: file
+        path: josm.sh
+
+      - type: file
+        path: org.openstreetmap.josm.appdata.xml
+
+      - type: file
```

Comment:
Are these not part of git repository you're cloning above?

Reply:
the git repository is not yet used, because java stuff doesn't build, easily. I envision that eventually the app will we built from source.